packages/fetch-api/src/api.ts

```diff
@@ -122,7 +122,7 @@ export class GiphyFetch {
      * @param options: SearchOptions
      * @returns {Promise<GifsResult>}
      **/
-    search(term: string, options: SearchOptions = {}): Promise<GifsResult> {
+    search(term: string, options: SearchOptions = {rating: 'pg-13'}): Promise<GifsResult> {
```

Comment:
is this kosher to do? or should i define a new type to specify the default value

Reply:
@kylebcarlson I think you want to go back to making `rating` optional in `option-types` and then you move this default value into the next line

```typescript
return request(`${getType(options)}/trending?${this.getQS({ rating: 'pg-13', ...options })}`, {
```
this ensures you have a rating of at least `pg-13` on every request, which I guess is the goal right?